webapp/javascript/pages/constants.ts

```diff
@@ -0,0 +1,16 @@
+export enum PAGES {
+  CONTINOUS_SINGLE_VIEW = '/',
+  COMPARISON_VIEW = '/comparison',
+  COMPARISON_DIFF_VIEW = '/comparison-diff',
+  SETTINGS = '/settings',
+  LOGIN = '/login',
+  SIGNUP = '/signup',
+  SERVICE_DISCOVERY = '/service-discovery',
+  ADHOC_SINGLE = '/adhoc-single',
+  ADHOC_COMPARISON = '/adhoc-comparison',
+  ADHOC_COMPARISON_DIFF = '/adhoc-comparison-diff',
+}
```

Comment:
thanks for doing this, this will definitely make it easier to work with these in the future.